history/subscriber/persister.py

```diff
@@ -155,6 +155,9 @@ def handle_event_data(self, tenant, message):
                 })
             if docs:
                 try:
+                    saved_timestamp = int(datetime.utcnow().timestamp() * 1000)
```

Comment:
Are there some reason to don't add the `saved_ts` together with the others attributes at document creation? In this case there is not performance impact, but I think the code become more cleaner, but it is *only a suggestion* feel free to accept or not.

Reply:
Hi @raulnegreiros, thank you for your feedback. Thare are a reason. That change is for performance tests, so we wanted to get timestamp just before saving de document or as close to it. We think to get timestamp inside "for" too, but we don't like to slow down performance.